app/backplane/radio_module/include/c_receiver_module.h

```diff
@@ -0,0 +1,64 @@
+#ifdef CONFIG_RADIO_MODULE_RECEIVER
+#ifndef C_RECEIVER_MODULE_H
+#define C_RECEIVER_MODULE_H
+
+#include "n_radio_module_types.h"
+#include "c_gnss_tenant.h"
+#include "c_udp_listener_tenant.h"
+#include "c_lora_transmit_tenant.h"
+#include "c_lora_to_udp_tenant.h"
+
+// F-Core Includes
+#include <f_core/c_project_configuration.h>
+#include <f_core/messaging/c_message_port.h>
+#include <f_core/os/c_task.h>
+#include <f_core/net/device/c_lora.h>
+
+class CReceiverModule : public CProjectConfiguration {
+public:
+    /**
+     * Constructor
+     */
+    CReceiverModule();
+
+    /**
+     * See parent docs
+     */
+    void AddTenantsToTasks() override;
+
+    /**
+     * See parent docs
+     */
+    void AddTasksToRtos() override;
+
+    /**
+     * See parent docs
+     */
+    void SetupCallbacks() override;
+
+private:
+    static constexpr const char* ipAddrStr = "10.2.1.1";
+    static constexpr uint16_t radioModuleSourcePort = 12000;
```

Comment:
Do we want like registry of name:port and/or name:ip address somewhere that all modules can see? That way we only have to keep track of them in one spot